packages/server-wallet/tsconfig.json

```diff
@@ -20,9 +20,9 @@
     {
       "path": "../client-api-schema"
     },
-    {
-      "path": "../nitro-protocol"
-    },
+    // {
```

Comment:
Do we have a github issue for switching over all packages to use the local `nitro-protocol`? It would be nice to flag this with a `TODO` and the github issue.

Reply:
It might even be worth adding a quick note about it to the readme if we're not going to address it soon.

Reply:
56dbf96